web/src/app/monitors/monitor-create-edit/monitor-create-edit.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
We don't haver spec files. We will introduce Cypress in the near future but not for the moment

Reply:
okay will remove this file.